src/main/java/uk/gov/digital/ho/hocs/workflow/client/caseworkclient/dto/AddressDto.java

```diff
@@ -6,7 +6,7 @@
 import lombok.Getter;
 import lombok.NoArgsConstructor;
 
-@AllArgsConstructor(access = AccessLevel.PRIVATE)
+@AllArgsConstructor()
```

Comment:
Are we using this anywhere? Can the change be reverted? I can't seen an instance where we use the `new AddressDto` object instantiation anywhere... I might have missed it.